src/templates/renderServiceTemplates.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const dots = require('dot');
+const crypto = require('crypto');
+
+/**
+ * Render templates for services
+ * @param {Config} config
+ * @returns {Promise<void>}
+ */
+async function renderServiceTemplates(config) {
+  const rpcpassword = crypto.randomBytes(12).toString('base64');
+  try {
```

Comment:
What is the point of global try/catch?

Reply:
I don't know, I thought it was good practice when fs operations are involved? Removed it now.